src/pudl/transform/eia861.py

```diff
@@ -2327,76 +2146,290 @@ def utility_data(tfr_dfs):
         how="outer",
     )
 
-    # Drop original operational_data_eia861 table from tfr_dfs
-    del tfr_dfs["utility_data_eia861"]
-
-    tfr_dfs["utility_data_nerc_eia861"] = transformed_ud_nerc
-    tfr_dfs["utility_data_rto_eia861"] = transformed_ud_rto
-    tfr_dfs["utility_data_misc_eia861"] = transformed_ud_misc
-
-    return tfr_dfs
+    return (
+        Output(
+            output_name="utility_data_nerc_eia861",
+            value=_post_process(transformed_ud_nerc),
+        ),
+        Output(
+            output_name="utility_data_rto_eia861",
+            value=_post_process(transformed_ud_rto),
+        ),
+        Output(
+            output_name="utility_data_misc_eia861",
+            value=_post_process(transformed_ud_misc),
```

Comment:
Is `output_name` needed in here to ensure that these `Output` objects are associated with the right elements described in `outs` in the `@asset` decorator?